### ISSUE TITLE
Use version ranges in lateral compatibility list

### ISSUE THREAD

**Issue by javier-godoy** [vaadin/directory#31]
Take for instance https://vaadin.com/directory/component/iron-icons

The addon has 4 released versions:
- Version 1.0.0 and 1.0,1,	which are compatible with Vaadin 10-13
- Version 2.0.0 and 2.0.1, which are compatible with Vaadin 14-23

The lateral list of supported versions is too long and obsolete versions of the framework (11-13, 15-21) take most of the space

![image](https://user-images.githubusercontent.com/11554739/219119728-5a3ea0ef-44a2-40d3-9143-1dc47b9e2a38.png)

The same information can be summarized as:
![image](https://user-images.githubusercontent.com/11554739/219119474-5d542776-2d93-4e07-a927-3f07eb4b4f4c.png)

**Comment by samie** [comment]
Good idea. This would be the same how the compatibility matrix works already.
